camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .attribute("enum.values", result.getEnumChoices(entry.getKey()))
+                        .detail(ResultErrorBuilder.errorAttribute("enum.values"), result.getEnumChoices(entry.getKey()))
```

Comment:
Maybe we can have a ResultErrorBuilder.detail(string, ...) ? so we can make it a little less verbose